pkg/resources/management.cattle.io/v3/setting/Setting.md

```diff
@@ -0,0 +1,19 @@
+## Validation Checks
+
+### Invalid Fields - Create
+
+When a Setting is created, the following checks take place:
+
+- If set, `disable-inactive-user-after` must be zero or a positive duration (e.g. `240h`).
+- If set, `delete-inactive-user-after` must be zero or a positive duration (e.g. `240h`).
```

Comment:
The `MinDeleteInactiveUserAfter` check does not seem to be documented. Is that intentional ?